tests/integration.sh

```diff
@@ -119,10 +119,19 @@ function package_offline_installer {
 function publishImage {
     echo "Publishing images to Docker Hub..."
     echo "The images on the host:"
+    # for master, will use 'dev' as the tag name
+    # for release-*, will use 'release-*-dev' as the tag name, like release-v1.8.0-dev
+    if [[ $DRONE_BRANCH == "master" ]]; then
```

Comment:
We should call the docker login first before pushing images

Reply:
do you mean line 133?